modules/identity/src/Volo.Abp.Identity.Domain/Volo/Abp/Identity/IdentityUserManager.cs

```diff
@@ -191,12 +190,13 @@ private async Task CheckMaxUserOrganizationUnitMembershipCountAsync(int requeste
         }
 
         [UnitOfWork]
-        public virtual async Task<List<OrganizationUnit>> GetOrganizationUnitsAsync(IdentityUser user)
+        public virtual async Task<List<OrganizationUnit>> GetOrganizationUnitsAsync(IdentityUser user, bool includeDetails = false)
         {
             await IdentityUserRepository.EnsureCollectionLoadedAsync(user, u => u.OrganizationUnits, CancellationTokenProvider.Token);
 
             return await OrganizationUnitRepository.GetListAsync(
                 user.OrganizationUnits.Select(t => t.OrganizationUnitId),
+                includeDetails,
                 cancellationToken: CancellationToken
```

Comment:
inconsistent usage of CancellationToken. CancellationTokenProvider.Token is used on EnsureCollectionLoadedAsync but CancellationToken is used here.